frontend/src/metabase/dashboard/containers/DashboardHeader.jsx

```diff
@@ -101,6 +100,8 @@ class DashboardHeader extends Component {
     setSidebar: PropTypes.func.isRequired,
     closeSidebar: PropTypes.func.isRequired,
     addActionToDashboard: PropTypes.func.isRequired,
+
+    databases: PropTypes.object,
```

Comment:
PropTypes.array?

Reply:
That's what I thought as well, but when I log the databases props, it's an object with the ID as the key, and the DB data as the value 
![Uploading image.png…]()